### DISCUSSION TITLE
Disabling PRs

### DISCUSSION THREAD

**Discussion by colocated** [renovatebot/renovate#31204] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Helloo,

I use Mend Renovate for all of my JS/other personal projects, and I've recently brought up the idea of using Renovate with our team's Ruby and Flutter/Dart projects at work.

They've mentioned they'd be interested in the dependency dashboard, but automatic PR creation is a feature they do **not** want at this point.

How do I install renovate and disable PR creation during the config PR?

Thanks,
Jack

### Logs (if relevant)

None

**Comment by RahulGautamSingh** [comment, marked as answer]
Try `dependencyDashboardApproval:true`

**Comment by enaluz** [comment]
Same issue for me too. Would appreciate some help! This is my current `renovate.json` file, but renovate bot still keeps making PRs.

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "assigneesFromCodeOwners": true,
  "automerge": false,
  "baseBranches": ["main"],
  "dependencyDashboard": true,
  "dependencyDashboardApproval": true,
  "dependencyDashboardTitle": "Renovate Dependency Dashboard",
  "draftPR": true,
  "extends": ["config:recommended", ":dependencyDashboard", ":dependencyDashboardApproval"],
  "ignoreDeps": [],
  "packageRules": [...]
}

```

**Reply by RahulGautamSingh** [reply]
I will close this discussionwith the above answer as it is correct.

For your case, please create a new discussion and also include the logs for the run in it.
